dashboard/app/views/levels/show.html.haml

```diff
@@ -25,7 +25,7 @@
       %h4{style: 'margin-left: auto; margin-right: auto; max-width: 800px;'}!= @level.properties['instructions']
   -# Pass special parameters to Blockly partial for embedded level type.
   = render partial: 'levels/blockly'
-- elsif @level.is_a?(DSLDefined) || @level.is_a?(Widget)
+- elsif @level.is_a?(DSLDefined) || @level.is_a?(Widget) || @level.is_a?(OpenResponse)
```

Comment:
this whole conditional is getting a bit hairy -- are there more meaningful names for these conditions (that might posslby be abstracted as a polymorphic method on Level?)